pages/common/wget.md

```diff
@@ -7,6 +7,10 @@
 
 `wget {{https://example.com/foo}}`
 
+- Mirror all listed files within a directory and its sub-directories only (no parent):
```

Comment:
Perhaps `sub-directories only, without mirroring any parent directories` would read better?

Reply:
I used the word "no parent" to emphasize the meaning of -np without actually having to expand to the TL;DR `--no-parent` switch name.

Reply:
@a-raccoon - Our aim here in tldr is to let people understand the command, rather than memorize the syntax. Please see my comment [here](https://github.com/tldr-pages/tldr/pull/1693#discussion_r153699659)